shaders/OrganelleOpaque.gdshader

```diff
@@ -4,11 +4,11 @@ render_mode depth_draw_opaque;
 uniform sampler2D texture : source_color;
 
 uniform sampler2D dissolveTexture : source_color;
-uniform float dissolveValue : hint_range(0, 1);
+uniform float dissolveValue : hint_range(0, 1) = 0.0;
+uniform float dissolveScale = 1.0;
```

Comment:
I assume the scale is 0-1 here? If so could you add a `hint_range` for this? That makes the value show up more intuitively in the Godot editor.

Reply:
1.0 is the default, but values above 1 decrease the size of the noise. Putting it like that, maybe I should have called it 'dissolveResolution'...
Values from 0.5 to 5.0 are fine.

Reply:
I think the name is clear enough, but you should add `hint_range(0.5, 5)` here to make the Godot editor show a slider for picking the value.